src/app/components/match/match.component.ts

```diff
@@ -35,44 +38,81 @@ export class MatchComponent implements OnInit {
               private router: Router) { }
 
   ngOnInit() {
-    this.volunteeringRequestsService.getAll().then(requests => { this.volunteeringRequests = requests; });
-    this.volunteeringOffersService.getAll().then(offers => { this.volunteeringOffers = offers; });
+    this.volunteeringRequestsService.getUnmatched().then(requests => {
+        this.volunteeringRequests = requests; });
```

Comment:
Either put them in the same line or put the closing`})` in another separate line.